amplify/backend/function/expressServer/src/controllers/portfolioController.js

```diff
@@ -160,8 +151,113 @@ catch (err) {
 }
 
 
+// Sell Stock Route
+const sellStockMarketOrder = async (req, res, next) => {
+  try{
+    // check that all of the data is there
+    if (
+      typeof req.body.portfolioId === 'undefined' ||
+      typeof req.body.stockId === 'undefined'||
+      typeof req.body.units === 'undefined'||
+      typeof req.body.buyOrSell === 'undefined'||
+      typeof req.body.orderType === 'undefined'
+    ) {
+      // data is missing bad request
+      res.status(400)
+      res.errormessage = 'Details missing for a transaction'
+      return next(
+        new Error(
+          'The client has not sent the required information to buy stock',
+        ),
```

Comment:
Only thing I could see was this. buy instead of sell

Reply:
Will fix this in the next update! Will merge now so that I can move onto the next steps. Thanks.